docs/source/api/v5/profileparameters.rst

```diff
@@ -175,14 +175,14 @@ Response Structure
 
 	{ "alerts": [
 		{
-			"text": "profileParameter was created.",
+			"text": "All Requested ProfileParameters were created.",
 			"level": "success"
 		}
 	],
 	"response": {
-		"lastUpdated": null,
-		"profile": null,
+		"lastUpdated": "2018-12-05T21:44:14.410503+05:30",
+		"profile": "testProfile1",
 		"profileId": 18,
-		"parameter": null,
+		"parameter": "testParam1",
 		"parameterId": 1
 	}}
```

Comment:
Since you are sending an array, it will return a response for both parameters being set to a profile. An eg as follows:
`{
	"alerts": [{
		"text": "All Requested ProfileParameters were created.",
		"level": "success"
	}],
	"response": [{
		"lastUpdated": "2023-08-23T11:02:33.584024-06:00",
		"profile": "test2",
		"profileId": 19,
		"parameter": "tm.toolname",
		"parameterId": 2
	}, {
		"lastUpdated": "2023-08-23T11:02:33.584024-06:00",
		"profile": "test2",
		"profileId": 19,
		"parameter": "maxRevalDurationDays",
		"parameterId": 3
	}]
}`

Reply:
Fixed.